apis/server/container_bridge.go

```diff
@@ -244,22 +244,13 @@ func (s *Server) getContainers(ctx context.Context, rw http.ResponseWriter, req
 			HostConfig: m.HostConfig,
 		}
 
-		// TODO encapsulate this into a single function
-		if m.State.Status == types.StatusRunning || m.State.Status == types.StatusPaused {
-			start, _ := time.Parse(utils.TimeLayout, m.State.StartedAt)
-			startAt, err := utils.FormatTimeInterval(start.UnixNano())
-			if err != nil {
-				return err
-			}
-
-			container.Status = "Up " + startAt
-			if m.State.Status == types.StatusPaused {
-				container.Status += "(paused)"
-			}
-		} else {
-			container.Status = string(m.State.Status)
+		status, err := m.FormatStatus()
```

Comment:
Move this part above above `container := types.Container{` and still in the for loop, then we can add status when constructing the instance, like:
```
		container := types.Container{
			ID:         m.ID,
			Names:      []string{m.Name},
			Image:      m.Config.Image,
			Command:    strings.Join(m.Config.Cmd, " "),
			Created:    t.UnixNano(),
			Labels:     m.Config.Labels,
			HostConfig: m.HostConfig,
            Status: status,
		}
```